ncclient/manager.py

```diff
@@ -158,6 +158,19 @@ def connect_tls(*args, **kwargs):
 
     return Manager(session, device_handler, **manager_params)
 
+def connect_UnixSocket(*args, **kwargs):
```

Comment:
@SpiffiRacoon -- can we rename this to `connect_uds`? Sorry I didn't notice this before, but it will be more consistent with `connect_ssh` and `connect_tls`.